Tutorials/Animations_with_xmovie.ipynb

```diff
@@ -0,0 +1,1515 @@
+{
```

Comment:
<p>Do we know what this warning is about? If we can't get it to go away, should we add an explanation, so people don't worry about it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/452/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>I add a warning ignore!</p>